README.md

```diff
@@ -58,15 +58,25 @@ All EP documents must be MIT licensed.
 ## Progress of an EP
 
 All EPs will be committed to the repository regardless of their acceptance.
-The initial status shall be **"DRAFT"**.
+The initial status shall either be **"INCUBATING"** or **"DRAFT"**.
+
+If the status is **"INCUBATING"** that means that it is expected that the
```

Comment:
@trevnorris this mistakenly got pushed as part of an update can you comment here instead:  https://github.com/nodejs/node-eps/pull/22